src/minter-contract-utils/src/evm_bridge.rs

```diff
@@ -50,6 +50,8 @@ pub struct EvmParams {
     pub next_block: u64,
     pub nonce: u64,
     pub gas_price: U256,
+    /// Last block to fetch logs from. If None, fetch until `Safe`
+    pub last_log_block: Option<u64>,
```

Comment:
The `EvmParams` are values queried from the evm canister. The `last_log_block` is not one of them, it should not be placed here.